Makefile.pre.in

```diff
@@ -617,6 +617,9 @@ all:		@DEF_MAKE_ALL_RULE@
 build_all:	check-clean-src $(BUILDPYTHON) platform sharedmods \
 		gdbhooks Programs/_testembed scripts checksharedmods rundsymutil
 
+.PHONY: all-with-ensurepip-dists-bundled
+all-with-ensurepip-dists-bundled: build_all bundle-ensurepip-dists
```

Comment:
The name is quite long, and I'm unsure what 'dists' means here:
```suggestion
.PHONY: all-with-bundled-ensurepip
all-with-bundled-ensurepip: build_all bundle-ensurepip
```

Reply:
> I'm unsure what 'dists' means

Basically, this refers to Python distribution packages. As in wheel zipfiles or sdist tarballs.
Historically, `ensurepip` shipped with two wheels — `pip` and `setuptools`. Some time ago, Pradyun removed the `setuptools` one, but the infrastructure for having an iterable of dists remained in the source code. Hence the plural form.

Also, this make target doesn't control whether the ensurepip module is included by itself. It only downloads the wheel into the in-module location. So I think implying that this bundles ensurepip in the make target name is inaccurate.

When Pradyun and I last discussed it, we agreed that the list of things is probably no longer needed. So I was going to have a follow-up PR to replace it with a singular var at some point. I didn't want to include refactoring in this patch — it would make it harder to manage a non-atomic PR. But I suppose that follow-up could change the target name to specifically say “pip” instead of “dists”.

FWIW, I had problems embedding this into the Makefile. So maybe the RMs would have other preferences.

Reply:
I chose to make use of another make target name suggestion. See: https://github.com/python/cpython/pull/12791#discussion_r1307596457. Marking this thread as resolved.